androidapp/app/src/main/java/com/example/lanweather/MainActivity.kt

```diff
@@ -6,14 +6,20 @@ import android.content.Context
 import android.os.Build
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
-import android.util.Log
-import android.widget.Button
 import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
 import androidx.navigation.findNavController;
 import androidx.navigation.ui.AppBarConfiguration
 import androidx.navigation.ui.setupWithNavController
+import com.example.lanweather.data.AppDatabase
+import com.example.lanweather.data.entity.SensorEntity
+import com.example.lanweather.data.model.Nws
 import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.squareup.moshi.JsonAdapter
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import java.io.IOException
+import com.squareup.moshi.*
```

Comment:
Do you think we can change this so that we only import the Moshi libraries that we need here?